session-02/exercise.js

```diff
@@ -17,6 +19,15 @@ function transformArrayToUpper(listOfStrings) {
 */
 function sumOfAllAges(listOfStudentObjects) {
   return;
```

Comment:
Your tests are failing because this empty return statement needs to be removed. Your code wont execute because this ends the function first.